packages/arb-ts/src/lib/bridge.ts

```diff
@@ -104,4 +109,258 @@ export class Bridge extends L2Bridge {
   public async getAndUpdateL1EthBalance() {
     return this.l1Bridge.getAndUpdateL1EthBalance()
   }
+
+  public async triggerL2ToL1Transaction(l2TransactionHash: string) {
```

Comment:
Couple things here, 

1) There could potentially be more `L2ToL1Transaction` in an L2 tx. Might make sense to have a convenience method that maps from tx hash to a particular `L2ToL1Transaction`, but there should be a separate one that starts with a specific event or an index and batch number I think

2) I think @fredlacs already mentioned, but having this block on the tx being redeemable doesn't really work since it could be a long time. I think splitting blocking logic into higher level code and having this just fail if it can't be redeemed yet would make sense